plasmicpkgs/antd/src/registerRate.ts

```diff
@@ -0,0 +1,65 @@
+import registerComponent, {
+  ComponentMeta,
+} from "@plasmicapp/host/registerComponent";
+import { Rate, RateProps } from "antd";
+import { Registerable } from "./registerable";
+
+
+export const rateMeta: ComponentMeta<RateProps> = { 
+name: "AntdRate",
+displayName: "Antd Rate",
+props: {
+    allowClear: {
+        type: "boolean",
+        description: "Whether to allow clear when click again",
+        defaultValue: true
```

Comment:
`defaultValueHint` instead of defaultValue, to avoid passing through unnecessary props when generating code